cmd/bmp180/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// bmp180 reads the current temperature and pressure from a BMP180.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/conn/pin"
+	"periph.io/x/periph/conn/pin/pinreg"
+	"periph.io/x/periph/devices"
+	"periph.io/x/periph/devices/bmp180"
+	"periph.io/x/periph/host"
+)
+
+func printPin(fn string, p pin.Pin) {
+	name, pos := pinreg.Position(p)
+	if name != "" {
+		log.Printf("  %-4s: %-10s found on header %s, #%d\n", fn, p, name, pos)
+	} else {
+		log.Printf("  %-4s: %-10s\n", fn, p)
+	}
+}
+
+func read(e devices.Environmental, loop bool) error {
+	for {
+		var env devices.Environment
+		if err := e.Sense(&env); err != nil {
+			return err
+		}
+		fmt.Printf("%8s %10s %9s\n", env.Temperature, env.Pressure, env.Humidity)
+		if !loop {
+			break
+		}
+		time.Sleep(100 * time.Millisecond)
```

Comment:
Since this is in a loop printing to stdout maybe it should be something slower that one can follow? Such as 2x per second?